pgml-extension/tests/test.sql

```diff
@@ -27,3 +27,4 @@ SELECT pgml.load_dataset('wine');
 \i examples/multi_classification.sql
 \i examples/regression.sql
 \i examples/vectors.sql
+\i examples/transformers.sql
```

Comment:
We don't run transformers or finetuning during tests, since they take a lot of RAM and download time.

Reply:
Removed them.